docs/cloud-workload-security/linux_expressions.md

```diff
@@ -68,6 +68,59 @@ Triggers are events that correspond to types of activity seen by the system. The
 | `unload_module` | Kernel | A kernel module was deleted | 7.35 |
 | `utimes` | File | Change file access/modification times | 7.27 |
 
+## FIM triggers
+
+In addition to regular triggers, `fim.write.file.*` fields allow to write rules that will fire
```

Comment:
```suggestion
In addition to regular triggers, `fim.write.file.*` fields allow to write rules that fire
```

Reply:
I applied all your suggestions to the template, thanks !